vivo/Dockerfile

```diff
@@ -23,8 +23,8 @@ RUN unzip VIVO.zip -d VIVO
 RUN rm VIVO.zip
 
 WORKDIR /usr/local/VIVO/
-COPY ./example-settings.xml example-settings.xml
-RUN mvn install -s example-settings.xml
+COPY ./settings.xml settings.xml
```

Comment:
Since the VIVO.zip contains an `example-settings.xml`, to avoid confusion, I would suggest removing it near line 23 above as well.

Reply:
I see now.  example-setting.xml is the filename in Vivo, like you say.  so this line should be reverted to  example-settings.xml